.circleci/config.yml

```diff
@@ -187,8 +187,8 @@ jobs:
           name: install android tools
           command: |
             sh packages/mobile/scripts/install-android-tools.sh
-            echo 'export PATH=$ANDROID_SDK_ROOT/tools/bin:$PATH'  >> $BASH_ENV
-            echo 'export PATH=$ANDROID_SDK_ROOT/tools:$PATH'  >> $BASH_ENV
+            echo 'export PATH=$ANDROID_SDK_ROOT/cmdline-tools/latest/bin:$PATH'  >> $BASH_ENV
```

Comment:
is this change needed here or part of the upgrade emulator and tools?

Reply:
This change was due to the folder changes with [cmdline tools](https://developer.android.com/studio#downloads) as opposed to the previously used download.